packages/earth-map/src/components/header/component.tsx

```diff
@@ -67,6 +69,15 @@ const Header = (props: IProps) => {
   const allInitiallySelected = group.length === allGroups.length;
   const [selectedGroups, setSelectedGroups] = useState(allInitiallySelected ? [] : group);
   const [dropdownState, setDropdownState] = useState('close');
+  const [availableOrgs, setAvailableOrgs] = useState([]);
+  const [isLoadingOrgsStats, setIsLoadingOrgsStats] = useState(true);
+
+  useEffect(() => {
+    fetchStats({ group: getAvailableOrgs(roles).join(',') }).then((res: any) => {
```

Comment:
We use async / await throughout the app, please use it here too.
Also use try / catch and and render the orgs from `getAvailableOrgs(roles)` in case the request fails.  This would allow the user to keep using the app